packages/hooks/src/hooks/useTokenBalance/useTokenBalance.ts

```diff
@@ -0,0 +1,45 @@
+import React, { useContext } from 'react';
+import { ethers } from 'ethers';
+import { Web3Context } from '../../Provider';
+import ERC20 from './ERC20.json';
+
+export const useTokenBalance = (address: string, owner: string) => {
+  const [isLoading, setIsLoading] = React.useState(true);
+  const [balance, setBalance] = React.useState<null | string>(null);
+  const [hasError, setHasError] = React.useState(false);
```

Comment:
maybe it's better to make this `errorMessage` and set the type to `string`. Then, depending on the error, we can pass a different message up